File: A2Z_striver/Cpp/Step1/Lec1/Forloop.cpp
// Nth Fibonacci Number https://www.naukri.com/code360/problems/nth-fibonacci-number_74156?utm_source=striver&utm_medium=website&utm_campaign=a_zcoursetuf
#include<bits/stdc++.h>
using namespace std;

int main()
{
    int j=0,k=1,m,n;
    cin>>n;
    if(n<=1){
        cout<<1;
    }
    else{
       for(int i = 1; i<n ;i++){
        m = j+k;
        j=k;
        k=m;
        }
    cout<<m;
    }   
}

File: A2Z_striver/Python/Step1/Lec1/Forloop.py
#Nth Fibonacci Number https://www.naukri.com/code360/problems/nth-fibonacci-number_74156?utm_source=striver&utm_medium=website&utm_campaign=a_zcoursetuf

j=0
k=1
m=0
n=int(input())

if n<=1:
    print(1)
else:
    for i in range(1,n):
        m = j+k
        j = k
        k = m
    print(m)
